docs/source/api/user-guide/transformations/aggregation-quantile.ipynb

```diff
@@ -1,374 +1,515 @@
+                "Individual quantiles can be estimated via `private_quantile`, which uses the exponential mechanism."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 5,
+            "metadata": {},
+            "outputs": [
+                {
+                    "data": {
+                        "text/plain": [
+                            "[6.0, 14.0, 29.0]"
+                        ]
+                    },
+                    "execution_count": 5,
+                    "metadata": {},
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "context = dp.Context.compositor(\n",
+                "    data=data,\n",
+                "    privacy_unit=dp.unit_of(contributions=1),\n",
+                "    privacy_loss=dp.loss_of(epsilon=1.),\n",
+                "    split_evenly_over=3,\n",
+                ")\n",
+                "\n",
+                "candidates = list(range(*bounds))\n",
+                "\n",
+                "def release_quantile(alpha):\n",
+                "    query = context.query().drop_null().private_quantile(candidates, alpha)\n",
+                "    return query.release()\n",
+                "\n",
+                "[release_quantile(alpha) for alpha in quartiles]"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "This is equivalent to the following, in the lower-level framework API:"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 6,
+            "metadata": {},
+            "outputs": [
+                {
+                    "data": {
+                        "text/plain": [
+                            "[6.0, 14.0, 29.0]"
+                        ]
+                    },
+                    "execution_count": 6,
+                    "metadata": {},
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "input_space = dp.vector_domain(dp.atom_domain(T=float)), dp.symmetric_distance()\n",
+                "\n",
+                "\n",
+                "def make_expo_quantiles(alphas, d_in, d_out):\n",
+                "    t_pre = dp.t.make_drop_null(*input_space)\n",
+                "    def make_from_scale(scale):\n",
+                "        return dp.c.make_basic_composition([\n",
+                "            dp.m.make_private_quantile(*t_pre.output_space, candidates, alpha, scale)\n",
+                "            for alpha in alphas\n",
+                "        ])\n",
+                "    return dp.binary_search_chain(make_from_scale, d_in=d_in, d_out=d_out)\n",
+                "\n",
+                "\n",
+                "m_expo_quantile = make_expo_quantiles(quartiles, d_in=1, d_out=1.)\n",
+                "m_expo_quantile(data)"
```

Comment:
There's an error in the notebook right now... but rather than fixing it, maybe drop this section, and just worry about the high level API?

Reply:
I don't actually see the error. Everything else is in framework API, so I wanted to keep things similar. This would be a good setting for RST tabs.